src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private Node<K, V>[] table;
+    private int defCapacity = 16;
+    private int capacity;
+    private int size = 0;
+    private double loadFactor = 0.75;
+    private int threshold = 0;
+
+    private void resize() {
```

Comment:
private methods should located after public ones since they have utility purpose